allensdk/brain_observatory/gaze_mapping/_gaze_mapper.py

```diff
@@ -0,0 +1,396 @@
+import numpy as np
+import pandas as pd
+
+from scipy.spatial.transform import Rotation
+
+
+class EyeTrackingRigObject(object):
+    """Class encompassing coordinate transforms on objects in the
+    eye tracking rig (camera, monitor).
+
+    Parameters
+    ----------
+    position_in_eye_coord_frame : numpy.ndarray
+        [x, y, z] position of the rig object in the eye coordinate system
+    rotations_in_self_coord_frame: numpy.ndarray
+        [x, y, z] rotations about the x, then y, then z axes to be applied
+        in the rig object's own coordinate system
+
+    """
+    def __init__(self,
+                 position_in_eye_coord_frame: np.ndarray,
+                 rotations_in_self_coord_frame: np.ndarray):
+        self._position = position_in_eye_coord_frame
+        self._rotations = rotations_in_self_coord_frame
+
+    @property
```

Comment:
I think that using properties and setters here is unncessary. There isn't any additional computation you need to hide behind the scenes with the property decorator; nor do any attributes depend on the value of other attributes. I would follow PEP-8 style here and recommend simply exposing these as public attributes.  https://www.python.org/dev/peps/pep-0008/#id49

Reply:
Making these all public seems reasonable, done!